### PR TITLE
Improve error handling

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: contentful/errors.py [hunk 1]
```diff
@@ -17,58 +20,141 @@ class HTTPError(Exception):
...
 
 
 class RateLimitExceededError(HTTPError):
     """
     429
     """
-    pass
+
+    def _default_error_message(self):
+        return "Rate limit exceeded. Too many requests."
```
Comment (mariobodemann) [inline contentful/errors.py]: Question for context: Can you parse the http headers here to include the time the user have to wait, or the current rate limit headers states?

**Review by mariobodemann — COMMENTED** [review]
Just one quick question for completeness, otherwise looks good to me ... :D

File: contentful/errors.py (hunk 1 above)
Comment (dlitvakb) [reply contentful/errors.py]: Yes, I can do that.

I'll add it.

Comment (dlitvakb) [reply contentful/errors.py]: Done

**Review by mariobodemann — APPROVED** [review]

**Review by mariobodemann — APPROVED** [review]
:ship:

**Pull request merged**